docs/guides/integration-options-private-operator.md

```diff
@@ -0,0 +1,77 @@
+
+A Cloud-based Private Operator runs in an <Link href="../ref-info/glossary-uid#gl-enclave">enclave</Link>&#8212;a memory-hardened virtual machine that prevents unauthorized access, so that unauthorized individuals, even if they are administrators, cannot download any configuration information or data from the virtual machine.
+
+This is an extra layer of security so that no one can see the salt values used to produce raw UID2s, and instead only the salt bucket ID is stored outside the Operator.
+
+Becoming a Private Operator includes several additional steps, and uses resources that the participant must provide.
+
+On this page, you'll find a high-level overview of integration steps and integration options, with links to additional information for each option.
+
+## Private Operator Requirements
+
+The participant must host, configure, maintain, and update the Private Operator instance, and must conform to strict security measures. Engineering resources are required to integrate and to make ongoing updates.
+
+The participant must sign a contract (see [Account Setup](../getting-started/gs-account-setup.md)) to host a Private Operator instance.
+
+:::note
+A Private Operator has no visibility into the raw UID2s or UID2 tokens processed by a Public Operator or another Private Operator. Each Private Operator is a completely closed infrastructure.
+:::
+
+## Private Operator Workflow
+
+Every Private Operator runs in an enclave (AWS), confidential space (GCP), or confidential computing environment, (Azure). Each of these ensures that the Private Operator runs in a protected memory space.
+
+The basic workflow is as follows:
+
+1. On startup, the Private Operator goes through an attestation process with the Core service. The attestation process verifies that the Operator is running in a secure and memory-hardened environment that is known and trusted, and that the environment hasn't been tampered with.
+
+1. When the Operator passes that attestation process, the Core service gives the Private Operator a secure S3 URL for retrieving the information it needs for startup.
+
+1. The Private Operator retrieves the security information from Amazon S3 that it needs to process UID2s, such as salts, encryption keys, and user opt-out records. For security details, see [Private Operator Security](#private-operator-security).
+
+1. If an Operator is restarted, it goes through the attestation process again, and retrieves a fresh set of security information.
+
+1. The Operator re-attests periodically with Core to ensure that it is still running in a protected environment. If any compromise is detected, the Operator shuts down.
+
+The information exchange between the UID2 Core service and the UID2 Opt-Out service is secured using TLS. (**GWH_TM01: is this actually part of the Private Operator?**)
```

Comment:
No, this is not necessary here. Suggest removing it.

Reply:
Done, thx.